aip/0194.md

```diff
@@ -0,0 +1,88 @@
+---
+aip:
+  id: 194
+  state: reviewing
+  created: 2019-08-16
+permalink: /194
+redirect_from:
+  - /0194
+---
+
+# Client-side retries
+
+This document aims to lay out guidance about which requests to retry, and for
```

Comment:
This paragraph still feels a bit odd. Can we change it to @jskeet's suggestion:

> RPCs sometimes fail. When one does, the client performing the RPC needs to know whether it is safe to retry the operation. When status codes are used consistently across multiple APIs, clients can respond to failures appropriately.

Reply:
Done